README.md

```diff
@@ -1,16 +1,15 @@
 # HELICS-SRC [![Build Status](https://travis-ci.org/GMLC-TDC/HELICS-src.svg?branch=master)](https://travis-ci.org/GMLC-TDC/HELICS-src) [![Build status](https://ci.appveyor.com/api/projects/status/afpa4mv0kgsjwvtn/branch/develop?svg=true)](https://ci.appveyor.com/project/nightlark/helics-src/branch/develop) [![Gitter chat](https://badges.gitter.im/GMLC-TDC/HELICS-src.png)](https://gitter.im/GMLC-TDC/HELICS-src) [![Documentation](https://img.shields.io/badge/docs-ready-blue.svg)](http://gmlc-tdc.github.io/HELICS-src)
 
 <p align="center">
-<img src="/images/HELICS_Logo.png" width="400">
+<img src="docs/img/HELICS_Logo.png" width="400">
 </p>
 
-
 Repository for the source code for the Hierarchical Engine for Large-scale Infrastructure Co-Simulation (HELICS) tool.
 
 # Install Instructions
 
-[Windows](./installing-on-windows.md)
+[Windows](docs/installing-on-windows.md)
```

Comment:
I made a similar change on master for cleaning up the root directory. Feel free to merge this change in, I'll fix the conflict when I am merging to master eventually.

Reply:
Lets merge this for now and then I will merge master into it before doing the PR into develop.  Then it should be clean when we eventually merge with Master.  Maybe next week for an alpha2 release.